src/main/java/utilities/UtilitiesFunctions.java

```diff
@@ -0,0 +1,42 @@
+package utilities;
+
+import managers.Context;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.Set;
+
+public class UtilitiesFunctions {
+
+    private static final Logger log = LogManager.getLogger(Logger.class.getName());
+
+    public static WebElement getWebElement(String xpath) {
+        return Context.driverManager.getDriver().findElement(By.xpath(xpath));
+    }
+
+    public static void switchToOtherTab() {
+        WebDriver driver = Context.driverManager.getDriver();
+
+        try {
+            Thread.sleep(2000);
```

Comment:
could you remove .sleep()?

Reply:
Again - I'd like to, but it simply doesn't work without it and I wasn't able to find any other solution for that. Do you have some ideas?

Reply:
if it doesn't work without sleeping thread there's something wrong with managing driver - maybe we can look at this problem more deeply?

Reply:
definitely :) a topic for the improvement